ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml

```diff
@@ -45,7 +45,7 @@ keys:
             description: Type 7 encrypted password.
           bfd:
             type: bool
-            default: False
+            default: true
```

Comment:
I think you changed the wrong one. This one should not be changed. And the other one was not changed.

Reply:
Ah yes
Sorry for this.